src/node/desktop/src/renderer/desktop-bridge.ts

```diff
@@ -91,9 +90,13 @@ export function getDesktopBridge() {
           }
 
           if (process.platform === 'win32') {
-            filePath = normalizeSeparatorsNative(filePath);
+            filePath = normalizeSeparatorsNative(filePath, '/');
```

Comment:
```suggestion
            filePath = normalizeSeparators(filePath, '/');
```

The name of the function, `normalizeSeparatorsNative()`, implies that it will always use the native separators for the current platform. If you want to use a specific separator, you should just use `normalizeSeparators()`.